install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -333,4 +333,13 @@ dnsConfig:
     {{- range .Values.global.podDNSSearchNamespaces }}
     - {{ render . }}
     {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+{{- if .Values.istio_cni.enabled }}
```

Comment:
IMO we should always add these values as annotations if they exist in the proxyConfig.

Reply:
Agree, I don't think there is anything wrong with additional annotations?

Reply:
Well from a quick check if we add the annotations to the pods for all cases then I have to modify 90% of the test cases.   I don't have a ton of  time in the next week due to kubecon so wondering if it is worth the effort.

Reply:
set `REFRESH_GOLDEN=1` in your env does it automatically if it is the tests I am thinking of. Just use with caution, don't blindly accept the output our the tests are really testing anything.

I don't care too much one way or another though